docs/get-started/Velox.md

```diff
@@ -153,31 +153,22 @@ mvn clean package -Pbackends-velox -Prss -Pspark-3.3 -DskipTests
 
 ## 2.3 HDFS support
 
-Hadoop hdfs support is ready via the [libhdfs3](https://github.com/apache/hawq/tree/master/depends/libhdfs3) library. The libhdfs3 provides native API for Hadoop I/O without the drawbacks of JNI. It also provides advanced authentatication like Kerberos based. Please note this library has serveral depedencies which may require extra installations on Driver and Worker node.
-On Ubuntu 20.04 the required depedencis are libiberty-dev, libxml2-dev, libkrb5-dev, libgsasl7-dev, libuuid1, uuid-dev. The packages can be installed via below command:
-```
-sudo apt install -y libiberty-dev libxml2-dev libkrb5-dev libgsasl7-dev libuuid1 uuid-dev
```

Comment:
Not necessary dependency?

Reply:
See this https://github.com/oap-project/gluten/blob/f7882f9d1478f9cc2a4381d69736a6ddb5372304/dev/package.sh#L18C54-L18C54

Reply:
thirdparty-lib will include all needed dependencies.